dlib/media/ffmpeg_demuxer.h

```diff
@@ -1457,15 +1523,12 @@ namespace dlib
         std::enable_if_t<is_image_type<image_type>::value, void>
         load_frame(image_type& image, const std::string& file_name)
         {
-            demuxer reader(file_name);
-            frame f;
-
-            if (!reader.is_open() || !reader.read(f) || !f.is_image())
+            if (!demuxer(image, video_enabled, audio_disabled).read(image))
```

Comment:
Haven't tested this yet but I bet it doesn't compile.
Need to write `demuxer({image, video_enabled, audio_disabled})` i'm pretty sure